src/components/CVE-2021-41773/CVE-2021-41773.tsx

```diff
@@ -3,6 +3,7 @@ import { useForm } from "@mantine/form";
 import { useCallback, useState, useEffect } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { UserGuide } from "../UserGuide/UserGuide";
```

Comment:
You do not need UserGuide as it is deprecated and unused. Please delete this line.